go/vt/discovery/healthcheck.go

```diff
@@ -301,6 +304,27 @@ type HealthCheckImpl struct {
 	healthCheckDialSem *semaphore.Weighted
 }
 
+// NewVTGateHealthCheckFilters returns healthcheck filters for vtgate.
```

Comment:
From looking at the binary flags, I can see that `--tablet_filters` can be supplied for `vtgate`, `vttablet`, and `vtctld`.  And `--keyspaces_to_watch can be supplied for `vtgate` and `vtctld`. And finally `--tablet-filter-tags` can be supplied for `vtgate` and `vtcombo`. So that answers this part.

Reply:
@mattlord sure, I can add those function comments

This block is a mostly-unchanged copy/paste of existing logic _(from `NewHealthCheck` - lower in the file)_, so the flag and functionality are at least partially documented already. Any suggestion what to add?

Reply:
I think you added the tablet filter flags? I'm not sure how it's supposed to work either as I've never seen any docs in the code or the website for it. 🙂 Maybe I missed it?